jobs/gtfs-rt-parser-v2/pyproject.toml

```diff
@@ -7,14 +7,14 @@ authors = ["Andrew Vaccaro <atvaccaro@gmail.com>"]
 [tool.poetry.dependencies]
 python = ">=3.8,<3.10"
 calitp = "2022.8.19a0"
```

Comment:
is there a strong reason not to bump this to try to keep it current? I'd kinda like everything to be on the latest calitp before we move too far into warehouse.